extension/src/prototypes/view/containers/Environments.tsx

```diff
@@ -48,12 +71,59 @@ const ambientOcclusionPropsAtom = atom((get): AmbientOcclusion => {
 export const Environments: FC = () => {
   const environmentType = useAtomValue(shareableEnvironmentTypeAtom);
   const colorMode = useAtomValue(colorModeAtom);
+  const showMapLabel = useAtomValue(showMapLabelAtom);
   const debugSphericalHarmonics = useAtomValue(debugSphericalHarmonicsAtom);
   const shadowProps = useAtomValue(shadowMapPropsAtom);
   const ambientOcclusionProps = useAtomValue(ambientOcclusionPropsAtom);
   const graphicsQuality = useAtomValue(shareableGraphicsQualityAtom) || undefined;
   const antialias = graphicsQuality === "ultra" ? "extreme" : graphicsQuality;
 
+  const styleOverrides: Record<ColorMode, any> = useMemo(
+    () => ({
+      light: {
+        default: {
+          fillColor: Color.BLACK,
+          outlineColor: Color.WHITE.withAlpha(0.8),
```

Comment:
We can't depend on Cesium, so could you update these definition to use a color code as string like `#000000` or `rgba(0, 0, 0, 0.8)`?

Reply:
For this part, we needed to convert the string color into Cesium's color, so I added this functionality in [this commit](https://github.com/reearth/reearth/pull/877/commits/adfc856de0c1591f63604dd0e171d7c89f8a0270) in reearth side.